plugins/org.locationtech.udig.project/src/org/locationtech/udig/project/internal/impl/StyleBlackboardImpl.java

```diff
@@ -596,10 +591,10 @@ public void putString(String key, String value) {
 
     @Override
     public void clear() {
-        if (content == null) {
-            return;
```

Comment:
well, if content is null, listeneres that might be registered already were never informed with the old implementation. Much better now!